src/index_runner/es_writer.py

```diff
@@ -39,9 +38,27 @@ def __init__(self):
                     global_mappings.update(_GLOBAL_MAPPINGS[g_map])
             self.init_index({
                 'name': index,
-                'alias': _ALIASES.get(index),
                 'props': {**mapping['properties'], **global_mappings}
             })
+        self.reload_aliases({})
+
+    def reload_aliases(self, data):
+        """Currently this function only adds new indexes to aliases
+        in future we want it to remove aliases that exist on elastic
+        but not in the config."""
+        if config()['global'].get('aliases'):
+            group_aliases = config()['global']['aliases']
+            print(f"Resetting Elasticsearch aliases....")
+            for alias_name in group_aliases:
+                try:
+                    _create_alias(
+                        f"{_PREFIX}.{alias_name}",
+                        [f"{_PREFIX }.{name}" for name in group_aliases[alias_name]]
+                    )
+                except Exception as err:
+                    names = group_aliases[alias_name]
+                    raise RuntimeError(f"alias name: {alias_name}, resulted in "
```

Comment:
Let's make this error message something like f"Failed creating alias {alias_name} for collections {names}". Also add `print(err)` here so we can actually see the error that was raised.